plugins/mapper-attachments/src/test/java/org/elasticsearch/mapper/attachments/MultifieldAttachmentMapperTests.java

```diff
@@ -153,4 +153,45 @@ public void testExternalValues() throws Exception {
         // In mapping we set store:true for suggest subfield
         assertThat(doc.rootDoc().getField("file.name.suggest").fieldType().stored(), is(true));
     }
+
+    public void testAllExternalValues() throws Exception {
+        String originalText = "This is an elasticsearch mapper attachment test.";
+        String forcedName = randomAsciiOfLength(20);
+        String forcedLanguage = randomAsciiOfLength(20);
+        String forcedContentType = randomAsciiOfLength(20);
+
+        String bytes = Base64.encodeBytes(originalText.getBytes(StandardCharsets.ISO_8859_1));
+        threadPool = new ThreadPool("testing-only");
```

Comment:
I'm fixing this. Sounds like a bad copy and paste which is not related to this PR.